### PR TITLE
Review Testing Documentation - 1st pass

### PR DESCRIPTION
**Pull request by d-g-h**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**
Updates testing documentation. Adds clarity.

**Steps necessary to review your pull request (required)**:
Proofread TESTING.md please, pretty please.

### CODE REVIEW

File: docs/TESTING.md [hunk 1]
```diff
@@ -1,11 +1,10 @@
-# Tests
+# Testing
 
 ## Test naming conventions
 
-- Use plain and proper english
-- Describe what the test is doing or checking
-- Don't re-mention component or page name as thats in the describe of the test
-- Use of should is not strictly enforced if if that doesn't make sense to describe the test in plain english
+- Use plain, and proper english
```
Comment (davidcarlsonberg) [inline docs/TESTING.md]: English should be capitalized.

File: docs/TESTING.md [hunk 2]
```diff
@@ -34,14 +33,17 @@ To develop in watch mode, please run
...
 npm run e2e:ci
 ```
 
-Check out the `.travis.yml` at root for actual implementation on Travis CI
+See [.travis.yml](https://github.com/infor-design/enterprise/blob/master/.travis.yml) for current implementation
 
-## Running BrowserStack Tests for Continuous Integration (CI) (WIP)
+## Running BrowserStack Tests on Travis Continuous Integration (CI) Server
 
 This will be ran in the evening (EST) in NYC, and tests <http://master-enterprise.demo.design.infor.com> by default
```
Comment (davidcarlsonberg) [inline docs/TESTING.md]: edit: This will run in the evening

File: docs/TESTING.md [hunk 3]
```diff
@@ -119,7 +123,7 @@ npx -n=--inspect-brk protractor test/protractor.local.debug.conf.js
 
 ## Working With Visual Regression Tests
 
-A visual regression test will be similar to the following code snippet. But because the tests run on ci in travis we need to mimic the screen shots on that environment. So you'll need to do some setup with docker (that will probably only work well on Mac).
+A visual regression test will be similar to the following code snippet. The tests run on Travis. Locally, in our development environment, we replicate the environment with Docker in order to capture, and compare screenshots on a nearly identical machine.  Below, we provide a guide for setup, and the generation of baseline images.
```
Comment (davidcarlsonberg) [inline docs/TESTING.md]: Remove comma after capture and after setup.

File: docs/TESTING.md [hunk 4]
```diff
@@ -146,17 +150,17 @@ if (utils.isChrome() && utils.isCI()) {
 }
 ```
 
-The next instructions are based on following [this guide](https://docs.travis-ci.com/user/common-build-problems/#Troubleshooting-Locally-in-a-Docker-Image) in order to debug Travis. We use the `node_js` [image](https://hub.docker.com/r/travisci/ci-nodejs/)
+Follow [this guide](https://docs.travis-ci.com/user/common-build-problems/#troubleshooting-locally-in-a-docker-image) in order to debug Travis. We currently use the `node_js` [image](https://hub.docker.com/r/travisci/ci-nodejs/)
 
-Many of the travis commands ran can be found in the [.travis.yml](https://github.com/infor-design/enterprise/blob/master/.travis.yml).
+Many of the travis commands ran can be found in the [.travis.yml](https://github.com/infor-design/enterprise/blob/master/.travis.yml), this will need to replicate inside of your container. This process is outlined below.
```
Comment (davidcarlsonberg) [inline docs/TESTING.md]: edit: travis run commands instead of travis commands ran

File: docs/TESTING.md [hunk 5]
```diff
@@ -146,17 +150,17 @@ if (utils.isChrome() && utils.isCI()) {
...
 
 ### Creating Baseline Screenshots
 
 1. Run `docker run --name travis-debug -dit travisci/ci-garnet:packer-1512502276-986baf0` to download the travis ci docker image to mimic the environment. And wait....
 1. Open up the image and go in `docker exec -it travis-debug bash -l`
 1. Switch to the travis user `su - travis`
 1. Go to home you home directory `(`cd ~`)`
```
Comment (davidcarlsonberg) [inline docs/TESTING.md]: edit: Go to your home directory

File: docs/TESTING.md [hunk 6]
```diff
@@ -170,17 +174,19 @@ wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
...
 1. Run the travis commands as per the build
 
 ```sh
-npm run quickstart & npm run e2e:ci
+npm run quickstart
+# In a new shell
+npm run e2e:ci
 ```
 
 1. Push the branch your working on to github and switch to the same branch on the vm
```
Comment (davidcarlsonberg) [inline docs/TESTING.md]: edit: Push the branch you're working on

**Comment by tmcconechy** [comment]
got mixed up a replaced with with bit https://github.com/infor-design/enterprise/pull/670
